rule-audit-client/jest.config.js

```diff
@@ -1,6 +1,8 @@
 module.exports = {
-  moduleFileExtensions: ['ts', 'tsx', 'js'],
+  moduleFileExtensions: ["ts", "tsx", "js"],
```

Comment:
Necessary to import fixtures, and ensure that imports relative to the 'src' directory are resolved correctly in tests.